CMakeLists.txt

```diff
@@ -3,7 +3,14 @@
 
 cmake_minimum_required(VERSION 3.8.0)
 
-# Fortran below is needed for SPHEREPACK.
+# Set up the project. Notes:
+# - Scripts may automatically regex-replace the version string in this command.
+#   For example, here's how you could set the version to 'X.Y.Z':
+#
+#   sed -i 's/^\(project(.*VERSION \)\([^ )]\)*/\1X.Y.Z/' CMakeLists.txt
+#
+#   Please don't break this command when editing this file.
```

Comment:
I don't understand what this comment is telling me to do.  When should I be setting the version number?

Reply:
clarified in a fixup